CondFormats/HcalObjects/interface/HcalQIEType.h

```diff
@@ -12,6 +12,7 @@ contains the QIE Typese + corresponding DetId
 class HcalQIEType
 {
  public:
+  enum {QIE8=0, QIE10=1, QIE11=2};
```

Comment:
I wasn't sure where to put it. Currently I use the enum in HcalDigitizer and HcalDbHardcode. Conceptually, the enum is used to make the HcalQIEType values human readable/understandable, but if there is a better/more standard location to define it, let me know.